src/App.tsx

```diff
@@ -14,11 +14,196 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+const lengths: number[] = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+
+interface State {
+  goods: string[];
+  isStarted: boolean;
+  isReversed: boolean;
+  sortBy: 'none' | 'name' | 'length';
+  selection: number;
+}
+
+class App extends React.Component<{}, State> {
+  state: State = {
+    goods: [...goodsFromServer],
+    isStarted: false,
+    isReversed: false,
+    sortBy: 'none',
```

Comment:
```suggestion
    sortBy: SortBy.none,
```
and for all strings along the code